server/container/cli/src/test/java/org/apache/james/cli/ServerCmdTest.java

```diff
@@ -1187,12 +1007,10 @@ public void createMailboxMappingCommandShouldThrowOnAdditionalArguments() throws
         String[] arguments = { "-h", "127.0.0.1", "-p", "9999", CmdType.CREATEMAILBOX.getCommand(), namespace, user, name, ADDITIONAL_ARGUMENT };
         CommandLine commandLine = ServerCmd.parseCommandLine(arguments);
 
-        control.replay();
         try {
             testee.executeCommandLine(commandLine);
         } finally {
-            control.verify();
-        }
+            }
```

Comment:
why do you keep the finally blocks ?

Reply:
my mistake, i used find & replace text to replace Easymock by mockito